packages/slate-hyperscript/test/selections/blurred-block.js

```diff
@@ -0,0 +1,51 @@
+/** @jsx h */
+
+import h from '../..'
+
+export const input = (
+  <value isFocused={false}>
```

Comment:
I think in keeping with JSX standards, the `isFocused` prop in hyperscript should be renamed to just `focused` instead.

Reply:
Fixed~